search/src/main/nodejs/search_create_default_channel.js

```diff
@@ -11,20 +11,24 @@ import {
   ALIGNMENT_JOB_COMPLETED
 } from "./awsappsyncutils";
 
-const searchBucket = process.env.SEARCH_BUCKET;
 const s3Retries = process.env.S3_RETRIES || 3;
 
 async function getSearchRecord(searchId) {
   const searchMetadata = await getSearchMetadata(searchId);
   return searchMetadata;
 }
 
+function getSearchBucket() {
+  return process.env.SEARCH_BUCKET;
+}
+
```

Comment:
Why are some env vars wrapped in a get function and others are not? I presume this is wrapped, so that it can be updated without having to redeploy the code. Is there no reason to do the same for the S3_RETRIES value?

Reply:
this function was mainly for testing. I could not get the proper env set in the before test method. If you have suggestions about how to make this available in the unit test I would prefer that.